scarlet/source.py

```diff
@@ -1,16 +1,17 @@
 from functools import partial
 
 from .initialization import *
-from .constraint import PositivityConstraint, MonotonicityConstraint, SymmetryConstraint, L0Constraint
-from .constraint import NormalizationConstraint, ConstraintChain, CenterOnConstraint
+from .spectrum import *
+from .morphology import *
+from .constraint import *
```

Comment:
Same as above. Class names in these modules cannot conflict.

Reply:
I get that there won't be a conflict, but it can be confusing to the reader. We already had a wildcard import in `source` and she was confused because she couldn't find a particular function anywhere in the code. If you really don't want to import all of the classes individually then you can import eg.`spectrum` and use `spectrum.Spectrum`.

Reply:
I see this as a case of user education. `source.py`, of all places, is where most of the code comes together. I'd rather make sure we have imported everything than writing a small sonnet to get the specific imports.

Reply:
I disagree. We should either import the module names or import the classes explicitly. At the very least you should define `__all__` so that the namespace isn't polluted, which is just a different sonnet.

Reply:
ok, ok